.github/workflows/image-health-check.yml

```diff
@@ -0,0 +1,90 @@
+name: Image Health Check
+
+env:
+  GH_TOKEN: "${{ secrets.GH_TOKEN }}"
+  PACKAGING_SECRET_KEY: "${{ secrets.PACKAGING_SECRET_KEY }}"
+  PACKAGE_ENCRYPTION_KEY: "${{ secrets.PACKAGE_ENCRYPTION_KEY }}"
+  PACKAGING_PASSPHRASE: "${{ secrets.PACKAGING_PASSPHRASE }}"
+  DOCKERHUB_PASSWORD: "${{ secrets.DOCKERHUB_PASSWORD }}"
+  DOCKERHUB_USER_NAME: "${{ secrets.DOCKERHUB_USER_NAME }}"
+  ACR_PASSWORD: "${{ secrets.ACR_PASSWORD }}"
+  ACR_SERVICE_PRINCIPAL_ID: "${{ secrets.ACR_SERVICE_PRINCIPAL_ID }}"
+  CURRENT_BRANCH: "${GITHUB_REF##*/}"
+  TEST: false
+on:
+  push:
+    branches: "**"
+  schedule:
+    - cron: "30 2 * * *"
+
+  workflow_dispatch:
+
+jobs:
+  check_docker_files_integrity:
+    name: Check if docker files are compliant with templates
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v2
+      - name: Update and check dockerfiles
+        run: |
+          ./update_dockerfiles
+          git add --intent-to-add dockerfiles
+          git diff --exit-code dockerfiles
+
+  build_package:
+    name: Build package
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        TARGET_PLATFORM:
+          - centos,8
+          - centos,7
+          - oraclelinux,8
+          - oraclelinux,7
+          - almalinux,9
+        POSTGRES_VERSION:
+          - 11
```

Comment:
Does this mean that we will try to create Citus packages for PG11? It is not clear to me what we are actually testing for when we run `test_citus_package.py` later in the file.